plugins/modules/net_tools/nmcli.py

```diff
@@ -345,12 +455,141 @@
+                             networks should be used with caution. In AP mode, the created network does not broadcast its SSID. Note that marking the network
+                             as hidden may be a privacy issue for you (in infrastructure mode) or client stations (in AP mode), as the explicit probe-scans are
+                             distinctly recognizable on the air.
+                type: bool
+                default: false
+            mac-address-blacklist:
+                description: A list of permanent MAC addresses of Wi-Fi devices to which this connection should never apply. Each MAC address should be given
+                             in the standard hex-digits-and-colons notation (eg "00:11:22:33:44:55").
+                type: list
+                elements: str
+            mac-address-randomization:
+                description: One of NM_SETTING_MAC_RANDOMIZATION_DEFAULT (0) (never randomize unless the user has set a global default to randomize and the
+                             supplicant supports randomization), NM_SETTING_MAC_RANDOMIZATION_NEVER (1) (never randomize the MAC address), or
+                             NM_SETTING_MAC_RANDOMIZATION_ALWAYS (2) (always randomize the MAC address). This property is deprecated for 'cloned-mac-address'.
+                type: int
+                default: 0
+                choices: [ 0, 1, 2 ]
+            mac-address:
+                description: If specified, this connection will only apply to the Wi-Fi device whose permanent MAC address matches. This property does not
+                             change the MAC address of the device (i.e. MAC spoofing).
+                type: str
+            mode:
+                description: Wi-Fi network mode; one of "infrastructure", "mesh", "adhoc" or "ap". If blank, infrastructure is assumed.
+                type: str
+                choices: [ infrastructure, mesh, adhoc, ap ]
+                default: infrastructure
+            mtu:
+                description: If non-zero, only transmit packets of the specified size or smaller, breaking larger packets up into multiple Ethernet frames.
+                type: int
+                default: 0
+            powersave:
+                description: One of NM_SETTING_WIRELESS_POWERSAVE_DISABLE (2) (disable Wi-Fi power saving), NM_SETTING_WIRELESS_POWERSAVE_ENABLE (3) (enable
+                             Wi-Fi power saving), NM_SETTING_WIRELESS_POWERSAVE_IGNORE (1) (don't touch currently configure setting) or
+                             NM_SETTING_WIRELESS_POWERSAVE_DEFAULT (0) (use the globally configured value). All other values are reserved.
+                type: int
+                default: 0
+                choices: [ 0, 1, 2, 3 ]
+            rate:
+                description: If non-zero, directs the device to only use the specified bitrate for communication with the access point. Units are in Kb/s, ie
```

Comment:
```suggestion
                description: If non-zero, directs the device to only use the specified bitrate for communication with the access point. Units are in Kb/s, so for example
```
Please don't use Latin abbreviations.